### DISCUSSION TITLE
First mention on Stack Overflow!

### DISCUSSION THREAD

**Discussion by mansellan** [twinbasic/twinbasic#310] [General]
https://stackoverflow.com/questions/68117796/which-classes-algorithms-in-system-security-cryptography-are-identical-to-those

> For a VB.NET-based VSTO add-in we need to plan for a future port to TwinBasic which is based on unmanaged COM architecture.

I guess twinBASIC will need a SO tag at some point 😄

**Comment by mwolfe02** [comment]
SO tags must be all lower-case but may contain hyphens.  That leaves two obvious options:

1. `twinbasic`
2. `twin-basic`

I'll post both options as separate comments to allow voting.

**Comment by mwolfe02** [comment]
`twinbasic`

**Comment by mwolfe02** [comment]
`twin-basic`

**Comment by DavidHollman** [comment]
Why would we need a hyphen? "twinBASIC" as @WaynePhillipsEA spells it has no spaces or punctuation. On SO hyphens in tags generally seem to stand for spaces.

**Comment by gregse99** [comment]
twinbasic - one word

**Comment by mwolfe02** [comment]
I just added the [`twinbasic` tag](https://stackoverflow.com/questions/tagged/twinbasic) to StackOverflow.  Another milestone complete.

**Reply by WaynePhillipsEA** [reply]
Awesome, thanks Mike!!

**Comment by mwolfe02** [comment]
I created a [tag wiki](https://stackoverflow.com/tags/twinbasic/info) for the twinBASIC tag at stackoverflow.  Feel free to edit it directly if you have the necessary SO reputation.  Otherwise, let me know here what you think should be changed.

**Reply by WaynePhillipsEA** [reply]
Thanks Mike, looks great!  I wouldn't have the reputation to edit, but looks fine anyway. Cheers
